packages/docs/docs/lambda/setup.md

```diff
@@ -82,37 +82,48 @@ Your package.json should look like the following:
 - Go to [AWS account IAM Users section](https://console.aws.amazon.com/iamv2/home#/users)
 - Click `Add users`
 - Enter any username, such as `remotion-user`.
-- **Check** the "Access key - Programmatic access" option.
-- **Don't check** the Management console access option. You don't need it.
-- Click "Next: Permissions", then "Next: Tags", then "Next: Review" without changing any settings.
-- Click "Create user", and ignore the warning that might appear.
-- Reveal the Secret access key.
-- Add a `.env` file to your project, and insert the following contents, using the credentials you just copied:
+- **Don't check** the "Enable console access" option. You don't need it.
+- Click "Next".
+- Click "Next" again without changing any settings. You should now be on the "Review and Create" step.
+- Click "Create user".
+
+## 5. Create an access key for the user
+
+- Go to [AWS account IAM Users section](https://console.aws.amazon.com/iamv2/home#/users)
+- Click on the name of the user that was created in step 4.
+- Navigate to the "Security Credentials" tab, and scroll down to the "Access Keys" section.
+- Click the "Create access key" button.
+- Select "Application running on an AWS compute service".
+- Ignore warnings that might appear and check the "I understand the recommendations..." checkbox.
+- Click "Next".
+- Click "Create access key".
+- Add a `.env` file to your project's root and add the credentials you just copied in the following format:
 
 ```txt title=".env"
 REMOTION_AWS_ACCESS_KEY_ID=<Access key ID>
 REMOTION_AWS_SECRET_ACCESS_KEY=<Secret access key>
 ```
 
-## 5. Add permissions to your user
+## 6. Add permissions to your user
 
 - Go to [AWS account IAM Users section](https://console.aws.amazon.com/iamv2/home#/users)
 - Select the user you just created.
-- Click "Add inline policy" on the right of the screen under "Permissions policies".
+- Click "Add inline policy" under the "Add Permissions" dropdown in the "Permissions policies" panel.
```

Comment:
> "right of the screen"

*Depends*. I was on a narrow version of AWS, so it defaulted to the mobile styles, and the option was no longer on the right. We should not add super-specific location specifications in the docs as it depends heavily on the layout and the viewport.

Also, the option was no longer on the right side of the screen 🙃